orchestration/src/main/java/com/sap/ai/sdk/orchestration/Message.java

```diff
@@ -49,16 +56,29 @@ static SystemMessage system(@Nonnull final String msg) {
     return new SystemMessage(msg);
   }
 
+  /**
+   * A convenience method to create a system message.
+   *
+   * @param msg the message content.
+   * @return the system message.
+   */
+  @Nonnull
+  static SystemMessage system(@Nonnull final MessageContent msg) {
+    return new SystemMessage(msg);
+  }
+
   /**
    * Converts the message to a serializable ChatMessage object.
    *
    * @return the corresponding {@code ChatMessage} object.
    */
   @Nonnull
   default ChatMessagesInner createChatMessage() {
-    if (this.getContent() instanceof MessageContentSingle) {
-      return ChatMessage.create().role(role()).content(content());
-    } else if (this.getContent() instanceof MessageContentMulti mCMulti) {
+    if (this.content() instanceof MessageContentSingle) {
```

Comment:
(Minor/Opinion)

I think it's a mistake to have this method on public API.
Maybe we could address this in another PR.